src/LSST/control/InputPortBits.cpp

```diff
@@ -48,7 +48,7 @@ void InputPortBits::writeBit(int pos, bool set) {
 
 
 bool InputPortBits::getBitAtPos(int pos) const {
-    if (pos < 0 || pos >= 32 ) {
+    if (pos < 0 || pos >= 32) {
```

Comment:
Instead of hardcoding the value of `32` or such kind of thing, I think it should be a constant that we could extend to 64bit or more when needed in the future. Such kind of value should be able to get programmatically based on the size of data type.

Reply:
Agreed. I've also been considering refactoring to use std::bitset, but I think that can wait.